rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -234,6 +234,13 @@
       );
     }
 
+    function hasExpression(text) {
```

Comment:
Can you change this name to `hasMathExpression`?

Reply:
Done